apps/nextjs-website/react-components/components/HighlightBox/HighlightBox.tsx

```diff
@@ -60,7 +60,7 @@ const HighlightBox = ({
               {body}
             </Typography>
 
-            {link && (
+            {link?.label && link?.href && (
```

Comment:
```suggestion
            {link && (
```
No need to check each field. If link exists both its fields exist.